cobalt/site/docs/development/setup-android.md

```diff
@@ -33,6 +33,8 @@ return and complete the following steps.
 
     If prompted, read and accept the license agreement.
 
+1.  Install [JDK 17](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html) and make sure it's before any other JDK on your PATH.
```

Comment:
Nit: I think it would make more sense to move this step to be the first or second in this section. All steps after Android Studio step are Cobalt customizations.

Reply:
moved